### PR TITLE
add epoch states

### PR DESCRIPTION
**Pull request by annamehr**

### CODE REVIEW

**Review by denniswell — APPROVED** [review]

File: docs/learn/epoch/index.md [hunk 1]
```diff
@@ -59,6 +59,18 @@ If not all orders can be executed, e.g. because there is not enough capital avai
 
 ## The Solver mechanism
```
Comment (armandomistral) [inline docs/learn/epoch/index.md:61]: @annamehr, suggesting to add a sentence defining the Solver mechanism here. This helps introducing the chapter, and prevents the doubling of headings...

File: docs/learn/epoch/index.md [hunk 2]
```diff
@@ -59,6 +59,18 @@ If not all orders can be executed, e.g. because there is not enough capital avai
...
 
+### How an epoch is currently displayed 
+These are the different states of an epoch.  
+
+| Status | Case | Consequence |
+| -------- | -------- | -------- |
+| Ongoing | Minimum epoch duration (24h) has not completed  | Minimum duration is passed (Investors can invest, redeem) |
```
Comment (armandomistral) [inline docs/learn/epoch/index.md:67]: - How about formatting the status label in bold? I.e. '**Ongoing**'
- Rather 'Minimum duration has passed'
- Rather '(Investors can invest and redeem)'

File: docs/learn/epoch/index.md [hunk 3]
```diff
@@ -59,6 +59,18 @@ If not all orders can be executed, e.g. because there is not enough capital avai
...
+| Minimum duration passed | No orders locked | Epoch will not close |
+| Minimum duration passed | Locked orders can be executed | Epoch closes automatically at 10AM CET |
```
Comment (armandomistral) [inline docs/learn/epoch/index.md:69]: - Why 10AM CET? Is this some kind of cron job?

File: docs/learn/epoch/index.md [hunk 4]
```diff
@@ -59,6 +59,18 @@ If not all orders can be executed, e.g. because there is not enough capital avai
...
+| Minimum duration passed | Locked orders can only be partially executed | Epoch needs to be closed manually |
+| Minimum duration passed | Locked orders cannot be executed | Epoch needs to be closed manually |
+| Computing orders | The epoch is being executed | New epoch is started |
```
Comment (armandomistral) [inline docs/learn/epoch/index.md:72]: - Rather 'New epoch has started'

**Review by armandomistral — COMMENTED** [review]
Thanks @annamehr. Added some suggestions…

**Pull request merged**